packages/components/tests/dummy/app/templates/components/form/checkbox.hbs

```diff
@@ -183,6 +183,18 @@
         <li>horizontal</li>
       </ol>
     </dd>
+    <dt>isRequired <code>boolean</code></dt>
+    <dd>
+      <p>Appends an indicator to the legend text and sets the
+        <code class="dummy-code">required</code>
+        attribute when user input is required.</p>
```

Comment:
I had to read the text multiple times to understand the role of the attribute. Maybe being mode specific makes it more obvious what happens behind the scenes, what receives that attribute.

```suggestion
        attribute on the controls when user input is required.</p>
```